src/database/migrations/1606158886329-CreateUserEntity.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateUserEntity1606158886329 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+    CREATE TABLE "user" (firstName varchar(255),
```

Comment:
We can discuss here what are the best types/amounts for each case & restrictions

Reply:
Is there a way of declaring these DB changes not using plain string and SQL? Just curious.

Reply:
https://typeorm.io/#/migrations/using-migration-api-to-write-migrations